### ISSUE TITLE
Results Page styling

### ISSUE THREAD

**Issue by wbshaw** [aunefyren/wrapperr#69]
Not so much of an issue, just a styling query. 
Is there any reason for the 15em Padding on all the "boks" divs - just adds huge gaps between sections and doesn't look tidy, 
Can't see any reason for it, but sure there probably is.

e.g.:

get_stats.js
114 >  117
`//INTRODUCTION
function load_introduction() {
    var text = "";
    text += "<div class='boks' data-color='red' style='width: 100%; padding-bottom: 15em; padding-top: 15em; height:auto; background-color:none'>";`

**Comment by aunefyren** [comment]
The purpose is to provide a more "full-screen" effect to each section and to let the color fade while scrolling. You feel it would be better to have a more compact page?

**Comment by wbshaw** [comment]
It's not too bad on mobile and flows ok, but on desktop it just feels like quite a big gap between sections, it's almost a full screen of space between each section and a lot of scrolling. I'm no designer though, so this is purely my opinion and I'm sure others may feel different.

**Comment by PhilipPetkov96** [comment]
I agree, I even added a scroll down message because initially It wasn't clear there is more on the page. Maybe arrows on the right side scrolling sections will be a good solution.
